forecasting/tests/unittests/test_forecasting.py

```diff
@@ -1,4 +1,54 @@
-import autogluon.forecasting
+import pandas as pd
+from autogluon.forecasting import ForecastingPredictor
+from autogluon.forecasting import TabularDataset
+import autogluon.core as ag
+
 
 def test_forecasting():
-	print("Not Implemented")
+    train_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/train.csv")
```

Comment:
Add the other tests we discussed

Reply:
Ideally the first test should not have any optional args specified (ie. what we expect most users to run), except what we need to ensure it runs quickly.  Is there any way for the test to run quickly without specifying: 
```
hyperparameters={
                                                                                 "MQCNN": {
                                                                                     'context_length': ag.Int(70, 90,
                                                                                                              default=prediction_length * 4),
                                                                                     "num_batches_per_epoch": 10,
                                                                                     "epochs": 5}
```
                                                           
Perhaps we can subsample the data to be even tinier and then the test will run quickly enough without any optional args like this?